Testing/Unit/Python/dummy_viewer.py

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python3
+
+import sys
+import os
+
+
+out = ""
+
+if len(sys.argv) > 1:
+    for x in sys.argv[1:]:
+        out = out + ' ' + x
+

```

Comment:
Instead of the loop:
" ".join(sys.argv[1:])

Reply:
Clearly my brain doesn't work in a Pythonic manner.

Reply:
No worries. I guess you're writing C++ in Python. My original R code looked like Python in R till Richard showed me the R way of doing things. Each language and its unique quirks.